jscomp/others/js_bigint.res

```diff
@@ -1,3 +1,11 @@
 /*** JavaScript BigInt API */
 
-type t
+type t = bigint
```

Comment:
Considering whether we should just remove this one and refer to `bigint` directly instead. It's breaking so we could wait for a bit before doing that and just deprecate `t` meanwhile, but we should definitely refer to `bigint` instead of `t` in the code below.

Reply:
Thanks! https://github.com/rescript-lang/rescript-compiler/pull/6670/commits/cf62db8067252acebd6eba665885b22080997bbc